docs/apis/ai-chat/api-reference.md

```
@@ -0,0 +1,477 @@
+    "limit": 10,
+    "offset": 0,
+    "hasMore": true
+  }
+}
+```
+
+## WebSocket API (Real-time)
+
+For real-time streaming interactions, connect to the WebSocket endpoint:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)